### PR TITLE
[Concept Entry] PyTorch : Parameter Updates

### LINKED ISSUE #5860: [Concept Entry] PyTorch : Parameter Updates
### Reason/inspiration (optional)

We would like a new entry on `Parameter Updates` concept under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/parameter-updates/parameter-updates.md`.       
  
The entry should include:      
  
- An introduction to the concept      
- A `Syntax` section that provides the syntax for the concept (if applicable)      
- An `Example` section that provides an example demonstrating the concept in use (if applicable)     

 
Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.|

### Entry Type (select all that apply)

New Concept Entry for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by mamtawardhani**
### Description
Added a Concept Entry for PyTorch : Parameter Update

### Issue Solved
Closes #5860 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/parameter-updates/parameter-updates.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+Title: 'Parameter Updates'
+Description: 'Parameter updates adjust model weights using an optimizer, which updates parameters based on gradients computed via backpropagation.'
+Subjects:
+  - 'Computer Science'
+  - 'Machine Learning'
+  - 'Deep Learning'
```
Comment (Radhika-okhade) [inline content/pytorch/concepts/parameter-updates/parameter-updates.md:7]:
```suggestion
  - 'Computer Science'
  - 'Deep Learning'
  - 'Machine Learning'
```

File: content/pytorch/concepts/parameter-updates/parameter-updates.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+- **Forward Pass**: Compute the model's predictions.
+- **Loss Calculation**: Measure how far the predictions are from the actual values.
+- **Backward Pass (Gradient Calculation)**: Compute gradients using `loss.backward()`.
+- **Parameter Update**: Adjust weights using an optimizer like `SGD` or `Adam`.
+- **Zero Gradients**: Clear accumulated gradients before the next iteration using `optimizer.zero_grad()`.
+
+## Example
+
+This code defines a simple neural network in PyTorch, computes the loss, performs backpropagation, updates the parameters using SGD, and prints the updated model weights:
```
Comment (Radhika-okhade) [inline content/pytorch/concepts/parameter-updates/parameter-updates.md:33]:
```suggestion
This code defines a simple neural network in PyTorch, computes the loss, performs backpropagation, updates the parameters using `SGD`, and prints the updated model weights:
```

File: content/pytorch/concepts/parameter-updates/parameter-updates.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+Title: 'Parameter Updates'
+Description: 'Parameter updates adjust model weights using an optimizer, which updates parameters based on gradients computed via backpropagation.'
+Subjects:
+  - 'Computer Science'
+  - 'Deep Learning'
+  - 'Machine Learning'
+Tags:
+  - 'Functions'
+  - 'Optimizers
+  - 'PyTorch''
```
Comment (Radhika-okhade) [inline content/pytorch/concepts/parameter-updates/parameter-updates.md:11]:
```suggestion
  - 'PyTorch'
```

**Comment by Radhika-okhade** [comment]
Thank you for contribuiting to Codecademy Docs @mamtawardhani.
